lava-v2-jobs-from-api.py

```diff
@@ -189,8 +186,9 @@ def main(args):
                                             modules_url = urlparse.urljoin(base_url, build['modules'])
                                         else:
                                             modules_url = None
-                                        if device['device_type'].startswith('qemu') or device['device_type'] == 'kvm':
-                                            device['device_type'] = 'qemu'
+                                        device_type = device['device_type']
+                                        if device_type.startswith('qemu') or device['device_type'] == 'kvm':
```

Comment:
Just noticed I was still using device['device_type'], will simplify that with device_type and update the commit.